cni-plugin/pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -67,9 +67,16 @@ func (d *linuxDataplane) DoNetworking(
 
 	d.logger.Infof("Setting the host side veth name to %s", hostVethName)
 
+	hostNlHandle, err := netlink.NewHandle()
+	if err != nil {
+		return "", "", fmt.Errorf("failed to create host netlink handle: %v", err)
+	}
+
+	defer hostNlHandle.Close()
```

Comment:
Is this an important detail of the performance fix, or just a better practice than we were doing before?

Reply:
Yes, this is the basis of the improvement. The idea is to create a netlink handle for the host namespace that we can use from the container network namespace, in order to run netlink commands on the host veth.

Reply:
Oh, I see, thanks.  So the point is that that keeps working to access the host namespace, even when inside the `ns.WithNetNSPath` call.